mmv1/template-converter.go

```diff
@@ -172,12 +172,7 @@ func convertHandwrittenFiles(folder, handwrittenTempFiles string) int {
 			log.Fatalf("Cannot open the file: %v", file)
 		}
 		data = replace(data)
-		goTemplate := ""
-		if strings.Contains(string(data), "{{") {
-			goTemplate = strings.Replace(file, ".erb", ".tmpl", 1)
-		} else {
-			goTemplate = strings.Replace(file, ".erb", "", 1)
-		}
+		goTemplate := strings.Replace(file, ".erb", ".tmpl", 1)
```

Comment:
making all files in templates/ a .tmpl file whether they are .erb or not. This simplifies the migration steps. We can always rename the .go.tmpl files back to a .go file later